rust/actix-web/src/middleware.rs

```diff
@@ -0,0 +1,135 @@
+use actix_web::body::EitherBody;
+use actix_web::body::MessageBody;
+use actix_web::dev::{forward_ready, Service, ServiceRequest, ServiceResponse, Transform};
+use actix_web::web::ReqData;
+use actix_web::{Error, Result};
+use actix_web_middleware_keycloak_auth::{
+    AlwaysPassPolicy, AuthError, DecodingKey, KeycloakAuth, KeycloakAuthMiddleware,
+    KeycloakAuthStatus,
+};
+
+const KEYCLOAK_PK: &str = "-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAvcOW0sCM9P9liqawHSd7wBi9L7bv42q+w7999HTBEWCX+1ofOwjSVTn//RPTbB42ak48QEtBqj8gBmr5KrFnpenF+sXbG7C4Qeef19n/0q6HjjrEK20695DE4y/a5QUdcxhepLGUmy55rDXS12oELfRJqD4HW6FqYnF0VSBAf6SxuR06de4toI5G2rPqfm2OveWLQel9JuHEte9D8IWJtG8va67XtwpHRIgRa+6CAHGFKWhZbwXOS55YuOL0er0XWnkdVJDTcdaPEvbvXfmQvHfP42JUrN0ZyIqyfo5X6XvfIEX5avLGvUdKd/9zKp4O9tL0Cg9OQWpXSNElquv+8wIDAQAB\n-----END PUBLIC KEY-----";
```

Comment:
This is copied and pasted from response JSON of http://${keycloak-service}:8080/realms/${realm_name}.